es6-promises/index.js

```diff
@@ -0,0 +1,10 @@
+const takeAChance = require('./take-a-chance');
+const nameProm = takeAChance('Arjun');
+nameProm.then(
+  value => {
+    console.log(value);
+  },
+  reason => {
+    console.error(reason);
+  }
+);
```

Comment:
Please make sure you only print the message part of the rejection reason, not the whole error stack trace.